src/components/PostInfo/PostInfo.jsx

```diff
@@ -1 +1,33 @@
-export const PostInfo = () => <>Put the post here</>;
+import { CommentList } from "../CommentList/CommentList";
+import { UserInfo } from "../UserInfo/UserInfo";
+
+
+export const PostInfo = ({ post }) =>
+  <div className="PostInfo">
+    <div className="PostInfo__header">
+      <h3 className="PostInfo__title">{post.title}</h3>
+
+      <p>
+        {' Posted by  '}
+
+        <UserInfo user={user} />
```

Comment:
user is not defined here
![Screenshot 2025-02-06 at 14 33 38](https://github.com/user-attachments/assets/f6e9ef5d-39b8-4bcb-bbd7-91aad65a84d0)